hazelcast/include/hazelcast/util/SyncHttpsClient.h

```diff
@@ -37,14 +37,16 @@ class HAZELCAST_API SyncHttpsClient
 public:
     SyncHttpsClient(const std::string& server_ip,
                     const std::string& uri_path,
-                    std::chrono::steady_clock::duration timeout);
+                    std::chrono::steady_clock::duration timeout,
+                    const std::string& secret_removal = std::string{});
```

Comment:
Reference to a temporary object???

Reply:
It is completely safe. [See ref.](https://en.cppreference.com/w/cpp/language/reference_initialization#Lifetime_of_a_temporary)

[An example](https://godbolt.org/z/zdrovvePT)